extensions/amp-auto-ads/0.1/placement.js

```diff
@@ -202,9 +203,9 @@ export class Placement {
   createAdElement_(baseAttributes) {
     const attributes = /** @type {!JsonObject} */ (Object.assign(dict({
       'layout': 'fixed-height',
-      'height': '0',
       'class': 'i-amphtml-layout-awaiting-size',
     }), baseAttributes, this.attributes_));
+    attributes['height'] = 0;
```

Comment:
is it intentional to use a number here?

Reply:
No it is not, I will change to string as before.